src/components/Label/__tests__/LobLabel.spec.js

```diff
@@ -39,7 +39,7 @@ describe('LobLabel', () => {
       const tooltip = getByTestId('tooltip-leading');
       expect(tooltip).toBeInTheDocument();
       const label = getByText(props.label);
-      expect(label.parentElement).not.toHaveClass('justify-between');
+      expect(label.parentElement.parentElement).not.toHaveClass('justify-between');
```

Comment:
Could we put a `data-testid` on that element instead of having to use `parentElement.parentElement` to target it?  That makes these tests brittle, just like you had to do here, if anyone moves anything around then these tests have to be updated and we shouldn't have to do that.